HISTORY.rst

```diff
@@ -8,6 +8,12 @@ Unreleased Changes
 ------------------
 * Updating primary repo url to Github.
 * Adding support for Python 3.8.
+* Removing the ``EncapsulatedOp`` abstract class. In practice the development
+  loop that encouraged the use of ``EncapsulatedOp`` is flawed and can lead to
+  design errors.
+* Removing unnecessary internal locks which will improve runtime performance of
+  processing many small Tasks.
+* Refactor to support separate TaskGraph objects that use the same database.

```

Comment:
I noticed that the `n_retries` parameter is removed from `Task.__init__()`.  Since this is a user-facing change, could you add a note here in `HISTORY` about it?

Reply:
Updated in 707625b